src/makeCalculator.js

```diff
@@ -37,7 +37,49 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const memory = {
```

Comment:
You don't really need to create variable here
But if you do like this, then check this point in checklist

<img width="775" alt="image" src="https://github.com/mate-academy/js_advanced_calculator/assets/86789161/758b5fd8-eeea-4a88-a295-9a5df2d22ada">